### PR TITLE
For PR - business search fetching active companies

### PR DESCRIPTION
**Pull request by jamespaologarcia**
*Issue #:* /bcgov/entity#17955

*Description of changes:*
Added extra filters for search results

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: package.json
Comment (severinbeauvais) [inline package.json:3]: You also need to run `npm i` and commit the package lock file.

File: src/services/business-lookup-services.ts
Comment (severinbeauvais) [inline src/services/business-lookup-services.ts:63]: This isn't ideal since the search returns a default number of entries and this might drop them all.

Try using the `status` parameter (as per line 36) where this is called from.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-723-3t8uvaue.web.app

File: src/services/business-lookup-services.ts
Comment (kzdev420) [reply src/services/business-lookup-services.ts:63]: Hi @severinbeauvais  @jamespaologarcia 

I checked the dev and local.
The dev site doesn't include the `state` query param when sending the request to the API
But local includes the `state` query on `feature-way-of-navigating` br. 

I am not sure why they are diff


Screenshot 2023-10-04 at 1 53 28 PM
Screenshot 2023-10-04 at 1 55 53 PM

Comment (severinbeauvais) [reply src/services/business-lookup-services.ts:63]: Are you searching for a business to create the same type of NR (eg, restoration)?

Comment (severinbeauvais) [reply src/services/business-lookup-services.ts:63]: Weird.

Comment (kzdev420) [reply src/services/business-lookup-services.ts:63]: Yes, I did same NR type

Comment (severinbeauvais) [reply src/services/business-lookup-services.ts:63]: Might have to do with current OCP issues related to authentication.

I do see here that the BusinessLookup _should_ be passed pass HISTORICAL or ACTIVE:
https://github.com/bcgov/namerequest/blob/07dff9c9ee9fee00ba291506523497957fc9921f/src/components/new-request/search-components/business-lookup-fetch.vue#L99
then
https://github.com/bcgov/namerequest/blob/07dff9c9ee9fee00ba291506523497957fc9921f/src/components/new-request/search-components/business-lookup-fetch.vue#L10
then
https://github.com/bcgov/namerequest/blob/07dff9c9ee9fee00ba291506523497957fc9921f/src/components/new-request/business-lookup.vue#L120

Comment (severinbeauvais) [reply src/services/business-lookup-services.ts:63]: James, you can explain it to the other reviewers.

I assume you're going to revert this code so I'm resolving this conversation.

Comment (severinbeauvais) [inline src/services/business-lookup-services.ts:63]: Remove this please :)

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-723-3t8uvaue.web.app

**Review by severinbeauvais — APPROVED** [review]

**Review by JazzarKarim — APPROVED** [review]
I've tried to fetch a historical business when doing a name change and tried to fetch an active business while doing a restoration. The businesses do not show up. Looks good James!

**Comment by JazzarKarim** [comment]
James, please we usually only merge once we get at least 2 approvals 😁

**Comment by jamespaologarcia** [comment]
> James, please we usually only merge once we get at least 2 approvals 😁

Noted. I'll keep that in mind next time. :)

**Pull request merged**
